reports/RJ722-II.md

```diff
@@ -0,0 +1,439 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/RahulJha
+student: Rahul Jha
+organisation: coala
+organisation_link : https://coala.io
+project: Improve Whitelisting Infrastructure for Vulture
+project_link: https://summerofcode.withgoogle.com/projects/#5620743946108928
+report_year: 2018
+tarball: https://drive.google.com/file/d/12HgBb2Wdjl6Zzm7mDqCt13ke8VHj7_Bb/view
+mentors: >
+ [Jendrik Seipp](https://github.com/jendrikseipp) & [Abdeali Kothari](https://github.com/AbdealiJK) & [Florian Bruhin](https://github.com/The-Compiler)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/28
+ - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/29
+ - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/30
+ - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/31
+bio: >
+ Rahul is currently a junior, studying Electronics Engineering at Zakir Hussain
+ College of Engineering and Technology, Aligarh Muslim University. He loves the
+ ideology behind Free and Open Source Software and reads about the history of
+ this revolutionary movement in his leisure time.
+social:
+ - GitHub:
+   - username: RJ722
+   - link: https://github.com/RJ722
+ - GitLab:
+   - username: RJ722
+   - link: https://gitlab.com/RJ722
+ - Gitter:
+   - username: RJ722
+   - link: https://gitter.im/RJ722
+ - Openhub:
+   - username: RJ722
+   - link: https://openhub.net/accounts/RJ722
+email: rj722@protonmail.com
+blog: https://rj722.github.io/blog
+activity:
+ - 1:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/b8a8ec8b42b24287560fd07ff3cafe30e92fb585
+   - details: Initial commit
+
+ - 2:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/43f6dae0818cdba691fb3f78e6bb4381867beeb3
+   - details: Create LICENSE
+
+ - 3:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/312313ad96e22386653ca39686a2eedfa00de82c
+   - details: Use README.rst and LICENSE.txt
+
+ - 4:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/d2fd1251e69e2e263645f9189c784a82fec81b4a
+   - details: Add .gitignore
+
+ - 5:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/22fbb2281503fc75b8960df528e878e2cfa24ed3
+   - details: Add docs and news
+
+ - 6:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/7fc63f7097ba83093740b4b3c1eea788fd485dd1
+   - details: Package codet
+
+ - 7:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/680cc58051d3640b73f9ae1ce936193843b9b604
+   - details: Ignore dist/
```

Comment:
give this a better description, as it does more than just `Ignore dist/` , and `Ignore dist/` should have been an automated part of setting up a repo

Reply:
I learned some important lessons like this one for setting up a new serious project. Would take care. Thanks for pointing this out!